OpenRA.Game/Player.cs

```diff
@@ -156,7 +156,7 @@ public Player(World world, Session.Client client, PlayerReference pr, MersenneTw
 			InternalName = pr.Name;
 			PlayerReference = pr;
 
-			inMissionMap = world.Map.Visibility.HasFlag(MapVisibility.MissionSelector);
+			inMissionMap = ((Map)world.Map).Visibility.HasFlag(MapVisibility.MissionSelector);
```

Comment:
no interface for this? IMissionSelector, Manifest.GetMissionSelector?

Reply:
Perhaps could be separated out but this is simply a flag for which map lists the map should appear on. This will be easy enough to do after the fact in a later PR once the main refactoring is complete. I think it is not needed for this PR which is already very large.